wallet/src/service/Offers.ts

```diff
@@ -41,6 +44,41 @@ type GiveOrWantEntries = {
   };
 };
 
+const CapDataShape = harden({ body: M.string(), slots: M.array() });
+
+const theOne = <T>(array: Array<T> | undefined) => {
+  assert(array?.length === 1);
+
+  const first = array.at(0);
```

Comment:
surprising to see `at`
```suggestion
  const first = array[0];
```

but actually, the first element could very well be undefined. so,
```suggestion
  return array[0];
```